components/form/builder/src/Standard/index.js

```diff
@@ -92,8 +94,34 @@ const checkConstrainstsFactory = json => ({for: fieldID, all}) => {
     if (checkboxShouldBeTrueConstraint && !checkboxValue)
       errorMessages = [checkboxShouldBeTrueConstraint.message, ...errorMessages]
   }
-
   return errorMessages
 }
 
+const checkConstrainstsFactory = json => ({for: fieldID, all}) => {
+  let fieldsToValidate = {}
+  if (all && fieldID) {
+    window.console.warn(
+      '[form/builder]: checkConstrainstsFactory: both modes validate all fields and validate a concrete field are not compatible, please use one of them'
+    )
+  } else if (all) {
+    fieldsToValidate = fieldsNamesInOrderOfDefinition(json?.form?.fields)
+  } else if (fieldID) {
+    fieldsToValidate = [fieldID]
+  } else {
+    window.console.warn(
+      '[form/builder]: checkConstrainstsFactory: Specify if you want to validate a specific field or all the fields'
+    )
+  }
+
+  let fieldsWithErrors = {}
```

Comment:
If you're going to use the reduce like this, better use a `forEach`.

Reply:
because forEach is fastest or more redeable?

Reply:
More readable and, in this case, avoid unnecesary transpilation with rest operator.